packages/modeling/src/utils/uniquePoints.js

```diff
@@ -0,0 +1,33 @@
+const geom2 = require("../geometries/geom2");
```

Comment:
@platypii Do you really think that this will be used anywhere else? If not then probably this should just be part of the hulls.

Also, I prefer function names with 'verbs' but not so picky. If you come up with a nice name then please change.

Reply:
I think it might be useful elsewhere, but for now the only places I saw were in the hulls.
So I just moved it into `hulls` directory.

I also gave it the verb name `toUniquePoints`. Open to suggestions if you have a better name.

Reply:
@platypii thanks. yeah. let's move toUniquePoints() out of hulls if there's another use case. i'm still searching...